pages/user guides/instance_recommendation.md

```diff
@@ -0,0 +1,64 @@
+---
+toc: false
+title: Cloud Instance Recommendations
+keywords: vm, ec2, instance, cloud, recommendation, cpu usage
+last_updated: September, 2024
+summary: "How to generate cloud instance recommendations"
+sidebar: main_sidebar
+folder: userguides
+permalink: instance-recommendation.html
+---
+
+Tidal Accelerator analyzes your servers and generates data-driven cloud instance recommendations which meet performance requirements for your workloads at the best possible value.
+
+Instance recommendation is currently available for Azure and Amazon Web Services.
+
+## How does it work?
+
+Tidal Accelerator's cloud instance recommendation uses a proven scientific approach to compare the CPU performance of your on-premise servers with their cloud compute equivalent. This allows us to recommend instances which will meet performance requirements for your workloads while delivering the best possible value, ensuring that workloads are not over-provisioned. 
+
+We consider CPU age and processing power, as well as RAM and storage usage to generate our recommendations. As you discover more about your server infrastructure, your instance recommendations change and become more refined, based on real-world usage data. 
+
+Under the hood, we leverage the approach described in Amazon Web Service's whitepaper <a target="_blank" href="https://s3.ca-central-1.amazonaws.com/tidal.assets/aws-demystifying-vcpus.pdf">Demystifying the Number of vCPUs for Optimal Workload Performance</a>.
+
+This scientifically proven methodology drives the CPU performance aspect of our recommendations.
+
+## Where do I start?
+
+### Projects
+
+Projects are how we determine the cloud provider used for instance recommendation. You can set the target cloud provider for a project when it is created, or you can update an existing project. You can also set cloud region at the project level. See [our documentation](assessment-projects.html#creating-a-project) for more information on creating a project.
+
+{% include image.html file="instance_rec_1.png" %}
+
+To assign servers to a project, you must create a migration wave. Simply create the migration wave for your project.
+
+{% include image.html file="instance_rec_2.png" %}
+
+After navigating to your migration wave, you can then add servers (and their dependencies, if required).
+
+All servers in this project will use the project's cloud provider and cloud region by default. If you need more granular control over cloud region, you can set this at the server level as well. Be aware that changing cloud provider or region at the project level will clear server-level cloud region settings, and re-generate recommendations for all servers in the project. 
+
+### Gather data
+
+After servers are assigned to a project, instance recommendation occurs automatically when your servers have enough data for us to begin making recommendations.
+
```

Comment:
There should be an info block here saying some thing like "The recommendation process sometimes can take several minutes to complete."